scala-generator/src/main/scala/models/generator/mock/MockClientGenerator.scala

```diff
@@ -45,14 +47,32 @@ object MockClientGenerator {
 
   }
 
+  object Http4s017 extends CodeGenerator {
+
+    override def invoke(form: InvocationForm): Either[Seq[String], Seq[File]] = {
+      val ssd = new ScalaService(form.service)
+      new MockClientGenerator(ssd, form.userAgent, ScalaClientMethodConfigs.Http4s017(ssd.namespaces.base, None)).invoke()
+    }
+
+  }
+
+  object Http4s018 extends CodeGenerator {
+
+    override def invoke(form: InvocationForm): Either[Seq[String], Seq[File]] = {
+      val ssd = new ScalaService(form.service)
+      new Http4s018MockClientGenerator(ssd, form.userAgent, ScalaClientMethodConfigs.Http4s018(ssd.namespaces.base, None)).invoke()
+    }
```

Comment:
Can't find where these are used

Reply:
I've used them in the test `Http4sClientMockGeneratorSpec` even though I noticed the "app" instantiates them differently.

Reply:
Changed the test to use the generator directly and removed this.